corehq/apps/app_manager/tasks.py

```diff
@@ -18,8 +18,8 @@
 @task(queue='background_queue', ignore_result=True)
 def create_usercases(domain_name):
     from corehq.apps.callcenter.sync_usercase import sync_usercase
-    for user in CommCareUser.by_domain(domain_name):
-        sync_usercase(user)
+    for user in CouchUser.by_domain(domain_name):
```

Comment:
this will include web users even if the FF is not turned on:
```suggestion
    if USH_WEB_USER_CASE_CREATION.enabled(domain_name):
        users = CouchUser.by_domain(domain_name)
    else:
        users = CommCareUser.by_domain(domain_name)
    for user in users:
```

(code changes require importing the toggle)